robottelo/ui/base.py

```diff
@@ -878,10 +879,20 @@ def sort_table_by_column(self, column_name):
         ]
         return cell_values
 
-    def perform_action_send_key_to_browser(self, key):
+    def perform_action_send_keys_to_browser(self, keys):
         """Send some key(s) to browser.
 
-        :param key: Key(s) to send to browser.
+        :param keys: Key(s) to send to browser. Keys may be several keys
+            including special, like ``control+shift+q``, single special or
+            simple key, e.g. ``escape``, ``q``.
         """
-        ActionChains(self.browser).send_keys(key).perform()
+        if '+' in keys:
+            keys_to_send = ''.join(
+                getattr(Keys, key.upper()) if key.upper() in dir(Keys) else
+                key for key in keys.split('+'))
```

Comment:
Will this implementation actually handle `control+shift+q` or it will rather press `control`, then `shift`, then `q` one by one independently?

Reply:
It will press control+shift+q simultaneously.